numpy/core/_add_newdocs.py

```diff
@@ -7969,62 +7969,198 @@ def luf(lamdaexpr, *args, **kwargs):
 #
 ##############################################################################
 
-add_newdoc('numpy.core.numerictypes', 'bool_',
-    """NumPy's Boolean type.  Character code: ``?``.  Alias: bool8""")
+def get_type(place, obj):
+    try:
+        return getattr(__import__(place, globals(), {}, [obj]), obj)
+    except Exception:
+        return None
```

Comment:
I think this should be `AttributeError`.

Reply:
Stolen from `add_newdoc` in `core/function_base.py`, where it also says `Exception`, but I've adapted it.